taskcat/cfn/template.py

```diff
@@ -170,6 +175,7 @@ def _find_children(self) -> None:
                 )
             self.children.append(child_template_instance)
 
+    @property
```

Comment:
How does this work when it has the property decorator? How are you going to pass in `template` ?

Reply:
this is a read-only property, so you'd never need to set it, or pass in a template. Example usage:

```python
t = Template(
    template_path=Path('~/quickstart-amazon-eks/templates/amazon-eks-master.template.yaml').expanduser().resolve() , 
    project_root=Path('~/quickstart-amazon-eks').expanduser().resolve()
)
for d in t.descendents:
    print(t.template_path)
```